ng2/5-piratenameservice/lib/src/name_service.dart

```diff
@@ -12,16 +12,31 @@ class NameService {
   static final Random _indexGen = new Random();
 
   final _names = <String>[
-    'Anne', 'Mary', 'Jack', 'Morgan', 'Roger',
-    'Bill', 'Ragnar', 'Ed', 'John', 'Jane'
+    'Anne',
```

Comment:
Fixed (while still using the formatter ... by putting a(n empty) comment at the end of the first line).